man/systemd.net-naming-scheme.xml

```diff
@@ -531,6 +531,20 @@
     particular version of systemd).</para>
   </refsect1>
 
+  <refsect1>
+    <title>Limit reading specific sysfs attributes</title>
+
+    <para>When creating names for network cards, some naming schemes use data from sysfs populated
+    by the kernel. This means that although a specific naming scheme in udev is picked,
+    the network card's name can still change when a new kernel version adds a new sysfs attribute.
+    Users can set an allowlist or denylist for these. To create an allowlist, the user needs to set
+    <varname>ID_NET_NAME_ALLOW=0</varname> for the device and then list the allowed attributes in the format
+    of <varname>ID_NET_NAME_SYSFSATTR=1</varname>. In case of a deny list, the user needs to set
+    <varname>ID_NET_NAME_ALLOW=1</varname> for the device and then list the denied attributes
+    in the format of <varname>ID_NET_NAME_SYSFSATTR=0</varname>. By default, all attributes are allowed.
+    </para>
+    </refsect1>
```

Comment:
Maybe mention how the property name is derived from a sysfs attr. name?

Reply:
Added an example